src/components/Exchange/FeesTooltip.tsx

```diff
@@ -68,46 +68,55 @@ function getFeesRows(isOpening: boolean, formattedFees: Record<string, string>)
   return rows.filter((row) => row.value);
 }
 
+function getTotalFees(fees: (BigNumberish | undefined)[]) {
+  return fees.reduce((acc: BigNumber, fee) => {
+    if (fee instanceof BigNumber) {
```

Comment:
`getTotalFees` function accepts list of BigNumberish (i.e. it could be string, number, BigNumber), but reducer checks if it's an instance of BigNumber

i guess either function should accept `(BigNumber | undefined)[]` instead, for reducer should check if it's not undefined instance of instance of BigNumber?

Reply:
Updated!